packages/client/modules/teamDashboard/components/AgendaItem/AgendaItem.tsx

```diff
@@ -109,32 +144,79 @@ const AgendaItem = (props: Props) => {
     ref.current && ref.current.scrollIntoView({behavior: 'smooth'})
   }, [])
 
+  useEffect(() => {
+    // if the tooltip has closed and we're not hovering in a new item, remove hover
+    // this is required because onMouseLeave isn't triggered if the cursor leaves the sidebar through the tooltip
+    if (tooltipStatus === PortalStatus.Exited && isHovering) {
+      updateHoveringId(null)
+    }
+  }, [tooltipStatus])
+
+  const handleIconClick = (e: React.MouseEvent) => {
+    e.stopPropagation()
+    UpdateAgendaItemMutation(
+      atmosphere,
+      {updatedAgendaItem: {id: agendaItemId, pinned: !pinned}},
+      {meetingId}
+    )
+  }
+
   const handleRemove = () => {
     RemoveAgendaItemMutation(atmosphere, {agendaItemId})
   }
+
+  const handleMouseMoveItem = () => {
+    // onMouseEnter isn't triggered if the cursor quickly moves over tooltip so check onMouseMove
+    if (!isHovering) {
+      updateHoveringId(agendaItemId)
+    }
+  }
+
+  const handleMouseMoveIcon = () => {
+    if (isHovering && tooltipStatus === PortalStatus.Exited) {
+      openTooltip()
+    }
+  }
+
+  const getIcon = () => {
+    if (pinned && isHovering) return <SvgIcon alt='unpinIcon' src={unpinIcon} />
+    else if (!pinned && !isHovering) return <Avatar hasBadge={false} picture={picture} size={24} />
+    else return <SvgIcon alt='pinnedIcon' src={pinIcon} />
+  }
+
   return (
-    <AgendaItemStyles title={content}>
-      <MeetingSubnavItem
-        label={content}
-        metaContent={
-          <AvatarBlock>
-            <Avatar hasBadge={false} picture={picture} size={24} />
-          </AvatarBlock>
-        }
-        isDisabled={isDisabled}
-        onClick={onClick}
-        isActive={isActive}
-        isComplete={isComplete}
-        isDragging={isDragging}
-        isUnsyncedFacilitatorStage={isUnsyncedFacilitatorStage}
-      />
-      <DeleteIconButton
-        aria-label={'Remove this agenda topic'}
-        icon='cancel'
-        onClick={handleRemove}
-        palette='midGray'
-      />
-    </AgendaItemStyles>
+    <>
+      <AgendaItemStyles onMouseMove={handleMouseMoveItem}>
+        <MeetingSubnavItem
+          label={content}
+          metaContent={
+            <IconBlock
+              onClick={handleIconClick}
+              onMouseMove={handleMouseMoveIcon}
```

Comment:
i went ahead & tried simplifying the hover & open logic on my own branch. 
i think the problem that you're bumping into is that when the mouse moves from item 1 to 2 it first passes over the tooltip, and then when the tooltip disappears, the mouse lands in the 2nd item without firing an OnMouseEnter event. 

as a hint, if you put a `pointEvents: 'none'` css prop on the tooltip, how can you simplify the hovering logic?

Reply:
Well, I'm never going to forget `pointerEvents: 'none'`... Thanks for your suggestion, Matt. It feels great to remove all of the additional complexity